curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-asynchronous-programming-by-building-an-fcc-forum-leaderboard/643c565936a10109b3f59326.md

```diff
@@ -16,13 +16,13 @@ Call the `showLatestPosts()` function at the end of your `try` block and pass in
 You should call `showLatestPosts()` at the end of your `try` block.
 
 ```js
-assert.match(code, /\s*console\.log\(data\);?\s*showLatestPosts\(.*\);??\s*/);
+assert.match(code, /\s*showLatestPosts\(.*\);??\s*/);
```

Comment:
I am not sure what these changes have to do with the issue you linked to this PR. 
Why is the `console` statement being removed from this test?
Is this a bug?

Reply:
My last commit was meant for https://github.com/freeCodeCamp/freeCodeCamp/pull/52961.

If it's acceptable, I'll revert it on this branch and put it where it needs to go.